fuel-client/src/client/schema/tx.rs

```diff
@@ -260,7 +262,7 @@ pub struct DryRunArg {
 )]
 pub struct DryRun {
     #[arguments(tx = &args.tx, utxo_validation = &args.utxo_validation)]
-    pub dry_run: Vec<OpaqueReceipt>,
+    pub dry_run: Vec<transparent_receipt::Receipt>,
```

Comment:
Why did you change the return type? (Sorry, I don't familiar with this code, so I wonder why we need changes here)

Reply:
https://github.com/FuelLabs/fuel-tx/pull/198#issuecomment-1273720252